sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,8 @@
+select  "g"."name" as "Genres",
+        count("g"."name")
+  from "genres" as "g"
+  join "filmGenre" using ("genreId")
+  join "castMembers" using ("filmId")
+  join "actors" using ("actorId")
+  where "actorId" = 178
```

Comment:
Assume the id for the actor can be different each time you run this, and only use the information you know up-front:

---

![image](https://user-images.githubusercontent.com/16143771/160179549-68229e08-adb2-4ed0-854f-0df7715ec930.png)